public/app/percona/shared/components/Elements/LinkTooltip/LinkTooltip.tsx

```diff
@@ -3,7 +3,8 @@ import { Icon, IconName, Tooltip, useTheme } from '@grafana/ui';
 import { getStyles } from './LinkTooltip.styles';
 
 export interface LinkTooltipProps {
-  tooltipText: string;
+  tooltipText?: string;
+  tooltipComponent?: React.Component;
```

Comment:
We can use just one prop for this. Maybe rename it to `tooltipContent: React.ReactNode`, which can be either text or a component.
Then, on render:
`{typeof tooltipContent === 'string' ? <span>tooltipContent</span> : tooltipContent}`

Reply:
Got it

Refactored